### PR TITLE
Reduced transition timings of questions in SinkToSwim minigame

### LINKED ISSUE #523: Sink to Swim Game slow question transition
Currently, when the skipbutton is pressed during the game the transition to next question is quite slow and the space in which the question is dispplayed remains blank foe ariund 2 seconds which is quite ambiguous.

slow

 Changed two lines of code to reduce the transition time.This could be reduced even further with slight adjustments in the start off and animation duration. 

smooth

@codingblazer @MeepyMay  Please check if this could be valid issue.

### PR DESCRIPTION
**Pull request by Rimjhim28**
### Description

Initially, the transition timings of questions in minigame SinkToSwim were quite slow. Through this PR, I have tries to reduce the animation transition timings to enhance the UX.

Fixes #523 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Refer to GIF here (Though it might appear a bit slow): 
https://user-images.githubusercontent.com/26908195/31807843-006472d8-b58e-11e7-994b-9d497b8eb98f.gif
Steps to Reproduce:
1. Press True button in SinkToSwim game
2. Press False button in SinkToSwim game
3. Press Skip button in SinkToSwim game

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Dilu9218** [comment]
@MeepyMay  The referred issue has been closed but this Pull Request has not been approved. If there is no reason I can approve this minor change.

File: PowerUp/app/src/main/java/powerup/systers/com/sink_to_swim_game/SinkToSwimGame.java [hunk 1]
```diff
@@ -208,9 +209,9 @@ public void answerChosen(View view) {
                 questionView.setBackground(getResources().getDrawable(R.drawable.swim_cross));
                 wrongAnswers++;
             }
+            questionView.setText("");
         }
 
-        questionView.setText("");
```
Comment (m-murad) [inline PowerUp/app/src/main/java/powerup/systers/com/sink_to_swim_game/SinkToSwimGame.java]: Why did you move this from here to inside of the `if` and `else if` blocks?

**Comment by Rimjhim28** [comment]
Before:
videotogif_2018 04 22_23 21 42

After:
after

@m-murad I am not sure if the change is noticable ehough but there is a slight time gap between fade out animation of text view and fade in animation of tick/cross which get covered up by moving ``` questionView.setText(""); ```
